vault/community.discord.channels.md

```diff
@@ -0,0 +1,149 @@
+---
+id: PIPLLg61HYfjK361B9Jm6
+title: Channels
+desc: ''
+updated: 1640815349912
+created: 1640809094657
+---
+
+![[community#join-other-dendrologists,1]]
```

Comment:
you can do a note ref to just a line. we don't need to reference everything, just the discord link